### DISCUSSION TITLE
vue3模板是不是还不支持编译成RN

### DISCUSSION THREAD

**Discussion by Thea1211** [NervJS/taro#15193] [Q & A]
vue3模板是不是还不支持编译成RN， taro自身是不是没法渲染显示的  没有类似render的api支持在web渲染显示的？

**Comment by ShaoGongBra** [comment]
你想他为啥叫RN React Native

**Comment by GreatAuk** [comment]
你怕是想要 taro vue3 -> taro react -> react native 吧 。。。

**Comment by zhiqingchen** [comment]
不支持
